### ISSUE TITLE
candidate for removal

### ISSUE THREAD

**Issue by jimdebeer** [vigour-io/example#8]
check with the native guys if this can go away
want to create one example thats called todoMVC which has all vigour features (better comparison)

**Comment by marcusbesjes** [comment]
I don't think this repo is in use right now, but I would keep this because its a good idea to have a general "blueprint" repo for your apps + dependencies / plugins + tool usage etc with documentation.

**Comment by nikoloza** [comment]
@jimdebeer I updated this repo and make it clean and usable to create new module (and also for new guys). So please keep it :)

https://github.com/vigour-io/example/pull/9
